src/NuGet.Core/NuGet.Configuration/Resources.resx

```diff
@@ -308,4 +308,7 @@
     <comment>0 - The name of the item
 1 - file path</comment>
   </data>
+  <data name="PackageNamespaceFound" xml:space="preserve">
+    <value>Package source namespace is found in nuget.config file.</value>
```

Comment:
The type for the config model had to contain some extra qualifiers, but for user facing messages, we want to refer to it as `Package namespace` or `Package namespaces`.

Reply:
This one is removed, but fixed for other places with same strings.